packages/itwin/grouping-mapping-widget/src/widget/components/GroupAction.tsx

```diff
@@ -91,6 +91,7 @@ export const GroupAction = (props: GroupActionProps) => {
     if (groups.length > 0) {
       if (showGroupColor) {
         await visualizeGroupColors(iModelConnection, groups, groups, hiddenGroupsIds, hilitedElementsQueryCache);
+        return;
```

Comment:
How is this related?

Reply:
Found this reset bug, it would `visualizeGroupColors` then ` clearEmphasizedElements` which was counterproductive. Not related, just forgot to make a change entry.

Reply:
Actually scratch that. I misremembered. The bug was fixed when I rewrote the entire checkbox logic. This was here because the old logic was breaking this reset call.

Reply:
Resolving as it's no longer relevant.